easybuild/framework/easyconfig/tweak.py

```diff
@@ -70,9 +70,10 @@ def ec_filename_for(path):
     return fn
 
 
-def tweak(easyconfigs, build_specs, modtool, targetdir=None):
+def tweak(easyconfigs, build_specs, modtool, targetdirs=None):
     """Tweak list of easyconfigs according to provided build specifications."""
-
+    if targetdirs is not None:
+        tweaked_ecs_path, tweaked_ecs_deps_path = targetdirs
```

Comment:
better initialise `tweaked_ecs_path` and `tweaked_ecs_deps_path` in case targetdirs is None, to avoid using undefined variables?